chance/p5js/chance-proximity/index.md

```diff
@@ -3,9 +3,9 @@ layout: p5js
 author: josues
 author-url: https://incom.org/profil/7029
 title: Law of proximity and chance
-tags: P5.js, chance, proximity, Law of proximity
+tags: [P5.js, chance, proximity, Law of proximity]
 permalink: /chance/chance-proximity/
 summary: This creates random proximities between ellipses.
 ---  
 
-![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-proximity/out.png)  
+![](./out.png)  
```

Comment:
Same as for the first sketch. The image is not needed for the site. It is more for the download. So people don't habe to run every sketch to see the output. The code of the index.js will be added to the page.